### PR TITLE
Made yaml a must for api files and add Zally hint

### PR DESCRIPTION
**Pull request by fmueller**
Related to #209

### CODE REVIEW

**Comment by maxim-tschumak** [comment]
:+1:

File: general-guidelines/GeneralGuidelines.md [hunk 1]
```diff
@@ -19,59 +19,65 @@ discipline that focus service interface design on...
...
+It is crucial to request and get early feedback — as early as possible, but not before the API changes are comprehensive
+with focus to the next evolution step and have a certain quality (including API Guideline compliance),
+already confirmed via team internal reviews.
 
 
 ## {{ book.must }} Provide API Reference Definition using OpenAPI
 
 We use the [OpenAPI specification](http://swagger.io/specification/) (aka Swagger spec) as standard for our REST API definitions.
-You should use YAML format (instead of JSON) for the OpenAPI API definition files due to its improved readability.
+You **have to use YAML format** (instead of JSON) for the OpenAPI API definition files due to its improved readability.
```
Comment (maxim-tschumak) [inline general-guidelines/GeneralGuidelines.md]: Maybe `have to ->provide<- the API definition in YAML format`? (doesn't matter which format is **used** during the design)

**Comment by fmueller** [comment]
Good point @maxim-tschumak I incorporated your change request.

**Comment by netme** [comment]
👍

**Comment by maxim-tschumak** [comment]
:+1:

**Comment by tkrop** [comment]
👍

**Pull request merged**